src/notifier.js

```diff
@@ -708,15 +699,22 @@ NotifierPrototype.uncaughtError = _wrapNotifierFn(function(message, url, lineNo,
     'useragent': navigator.userAgent
   };
 
-  var payload = this._buildPayload(new Date(), this.options.uncaughtErrorLevel, message, stack);
-  this._enqueuePayload(payload, true, [this.options.uncaughtErrorLevel, message, url, lineNo, colNo, err]);
+  var payload = this._buildPayload(new Date(), this.options.uncaughtErrorLevel,
+    message, stack, context);
+  this._enqueuePayload(payload, true, [this.options.uncaughtErrorLevel,
+    message, url, lineNo, colNo, err]);
 });
 
 
 NotifierPrototype.global = _wrapNotifierFn(function(options) {
   options = options || {};
+  var knownOptions = {
+    startTime: options.startTime,
+    maxItems: options.maxItems,
+    itemsPerMinute: options.itemsPerMinute
+  };
 
-  Util.merge(window._globalRollbarOptions, options);
+  extend(true, window._globalRollbarOptions, knownOptions);
```

Comment:
Are there any keys that a user could have put into global options, which would have had an effect?

Based on the docs, it appears that the answer is no, but I want to make sure I'm not missing a way that this could be backwards-incompatible.